swap_meet/item.py

```diff
@@ -1,2 +1,45 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__ (self, category="", condition = 0):
+        self.category = category
+        self.condition = condition
+    
+    def __str__(self):
+        return f"Hello World!"
+
+    def condition_description(self):
+        if self.condition < 1:
```

Comment:
This works perfectly well, but I thought I'd mention an alternative strategy! Explicitly writing this conditional logic does give us a lot of control over exactly under which situations a particular description should be used, but especially when we have a long list of chained checks that are very similar, we could try coming up with a way to represent it with data and access it by index. A basic example of this that would mirror your existing behavior could look like:
```python
# outside class definition
DESCRIPTIONS = (
    "Worn down",
    "Heavily Used",
    "Gently Used",
    "Solid",
    "Practically perfect",
    "Never worn before"
)

# in Item class definition
    def condition_description(self):
        return DESCRIPTIONS[self.condition]
```
We would still want to think about the edge case behaviors (outside the expected range? non-integer conditions, etc.), but we would have more general confidence that if this logic works for a particular condition, it's likely to work for all of them.